### ISSUE TITLE
Add Events Of Interest

### ISSUE THREAD

**Issue by MattSchaeffer** [MattSchaeffer/EventFiend#2]
On load I want it search for a group of event, that if it finds, it displays.  This will include (but isn't limited to):

- Last Reboot
- 7034 (service terminated Unexpectedly
- 1001 (BSOD)
- 6008 (Dirty shutdown)

**Comment by ikluhsman** [comment]
I'd like to be able to think up my own events of interest on the fly and filter for them. Usually, when I'm troubleshooting a windows based issue using event logs, I'll need to filter particular events that I'm interested in depending on the windows application/sub-system. I'd like all of the events on a system to be available to me in this tool for that reason, rather than what the developer of this app thinks is important for me to find.

I think that there should be a slightly different way of choosing event ID's to filter for, like just a plain comma seperated list I can input that allows me to filter for whatever Event ID's I want, kind of like the windows event viewer console's filter mechanism.

If you want to guide someone on what you think the appropriate events are for particular applications like Active Directory events, or Windows Update Events, or security events, etc then a more appropriate place for this might be a pop-up that is generated from the "Help" menu that has some guidance on what you think important events are. Then let the user of the app filter for those themselves. But don't box me in here, I need access to ALL the events on a system, it only makes sense.

If I can't decide for myself what events I want to see, and the developer is deciding what's important to filter in, then the tool becomes too rigid and less useful.

**Comment by MattSchaeffer** [comment]
I agree.  I was using it just today, and that really bugged me that I couldn't input events that I was looking for.  I think that I need an input, like you suggested, a control that shows which logs/events are being searched, and I need to turn the existing menu into  a reference that just adds those events to the list being searched as a quick helper if you don't know the events you are wanting., rather than a search of it's own.  That whole section was an afterthought, rather than something I had planned, so it wasn't really the best implementation
